bootstrap/src/main/java/se/fortnox/reactivewizard/Main.java

```diff
@@ -24,7 +24,6 @@ public static void main(String[] args) {
                 @Override
                 protected void configure() {
                     bind(String[].class).annotatedWith(Names.named("args")).toInstance(args);
-                    bind(ConfigFactory.class).toInstance(configFactory);
```

Comment:
Without this, won't all AutoBindModules will get a ConfigFactory that is not instantiated correctly?

Reply:
I dont think so. I tested to set a breakpoint in the constructor and it loads the config file because the @Named("args") String[] args is bound. db queries seems to work properly indicating that the configuration is loaded properly and changing logging properties takes effect as well.